pkgs/development/mobile/androidenv/tools.nix

```diff
@@ -6,7 +6,7 @@ deployAndroidPackage {
   nativeBuildInputs = [ makeWrapper ]
     ++ lib.optionals (os == "linux") [ autoPatchelfHook ];
   buildInputs = lib.optional (os == "linux") (
-      (with pkgs; [ glibc freetype fontconfig fontconfig.lib])
+      (with pkgs; [ glibc freetype fontconfig fontconfig.lib stdenv.cc.cc ])
```

Comment:
```suggestion
      (with pkgs; [ glibc freetype fontconfig fontconfig.lib
         stdenv.cc.cc # fix for https://github.com/NixOS/nixpkgs/issues/226357
      ])
```

Reply:
applied in https://github.com/NixOS/nixpkgs/pull/226489/commits/575e9aad09b82b10e28d0efd34413b2c685481da